tests/InputFiles/ExampleExecutables/RandomAmr1D.yaml

```diff
@@ -25,6 +25,7 @@ Amr:
         MaximumRefinementLevel: 8
   Policies:
     Isotropy: Anisotropic
+    RefinementLimits: Auto
```

Comment:
Set max refinement level to 8?

Also, I think you can delete the max refinement level from the `Random` criterion in this PR, should be very quick to do.